src/img/octicons/octicons.css

```diff
@@ -0,0 +1,221 @@
+@font-face {
```

Comment:
Consider linking the CDN in `index.html` instead of installing all the font files!